doc/Core.xml

```diff
@@ -6668,6 +6756,26 @@ Event description:
     <tt:ElementItemDescription Name="CurrentLocation" Type=“tt:GeoLocation"/>
   </tt:Data>
 </tt:MessageDescription>
+]]></programlisting>
+      </section>
+      <section>
+        <title>Cloud Firmware Upgrade</title>
+        <para>If a device supports clod firmware upgrade, it should provide the cloud firmware
```

Comment:
typo clod -> cloud

Reply:
Added a suggestion for the typo in my review for is selection.